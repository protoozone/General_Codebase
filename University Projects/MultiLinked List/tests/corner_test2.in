NEW 2
lol
23
INSERT 0 0       
INSERT 0 0  lol this is the string ig
INSERT 0 1 { 90}
INSERT 0 1 {90 }
INSERT 0 1 {}
INSERT 0 1 {invalid}
INSERT invalid 1
INSERT 0 invalid
VIEW ALL OBJECTS
VIEW 0 ELEMENT
REMOVE ELEMENT 0
PLEASE
NEW
VIEW
VIEW-NESTED
INSERT
REMOVE
DELETE
INSERT 0 0 {-23}
NEW -2
VIEW -2
VIEW-NESTED -2
REMOVE -2
INSERT -2 0 lol
INSERT 0 -2 lol
DELETE -2 0
DELETE 0 -2
NEW lol
VIEW lol
VIEW-NESTED lol
INSERT lol 0
DELETE lol 0
REMOVE lol
EXIT