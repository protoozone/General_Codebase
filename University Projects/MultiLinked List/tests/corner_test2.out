List 0: lol -> 23
List 0:       -> lol -> 23
List 0: lol this is the string ig  ->       -> lol -> 23
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
Number of lists: 1
List 0
lol this is the string ig  ->       -> lol -> 23
INVALID COMMAND: REMOVE
INVALID COMMAND: INPUT
INVALID COMMAND: NEW
INVALID COMMAND: VIEW
INVALID COMMAND: DELETE
INVALID COMMAND: INSERT
INVALID COMMAND: REMOVE
INVALID COMMAND: DELETE
List 0: {-23} -> lol this is the string ig  ->       -> lol -> 23
INVALID COMMAND: NEW
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW-NESTED
INVALID COMMAND: REMOVE
INVALID COMMAND: INSERT
List 0: {-23} -> lol this is the string ig  ->       -> lol -> lol -> 23
INVALID COMMAND: DELETE
List 0: {-23} -> lol this is the string ig  ->       -> lol -> 23
INVALID COMMAND: NEW
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW-NESTED
INVALID COMMAND: INSERT
INVALID COMMAND: DELETE
INVALID COMMAND: REMOVE